### PR TITLE
Update Documentation

### PR DESCRIPTION
**Pull request by ja0911**
Closes [#14](https://github.com/Code-4-Community/vsc/issues/14).

The following updates were made to the README.md: 

- [x] The structure of the project at a high level
- [x] Commands you can run (npm run ___)
- [x] Dev setup
- [x] Description of the project
- [x] Emojis
- [x] Links to the C4C website and LBFE
- [x] Removed any instances of ```yarn```

We also deleted `/pages/api/hello.ts` as well as its test `/test/hello.test.ts` to clean up the repository of default files as we were going through the directory.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,13 +98,25 @@ Work in Progress, will finish when we solidify our project structure.
...
 │   └── index.tsx
 ├── prisma       # Prisma DB Schemas, Prisma Client, Migrations, etc.
 │   ├── migrations
 │   │   ├── 20211223003440_test
 │   │   │   └── migration.sql
+│   │   ├── 20220129153613_added_news_entity
+│   │   │   └── migration.sql
+│   │   ├── 20211223003440_test
+│   │   │   └── migration.sql
 │   │   └── migration_lock.toml
```
Comment (rymaju) [inline README.md:120]: ⛏️ I wonder if we need to include these? Not sure it tells you much really. I would lean in favor of deleting them and just keeping /migrations

File: README.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
 # 🏡 Virtual Senior Center
 
-A project to create a virtual community for LGBT seniors in the Massachusetts area.
+The mission of the outstandinglife.org is to improve the quality of life of LGBTQ older adults in Massachusetts. Through this project, C4C aims to help LBFE find new creative ways to connect older adults: engaging hearts, transforming lives and breaking down boundaries. 
+
+(To be updated when website is deployed)
+Website: [Outstanding Life](https://outstandinglife.org/)
 
 ## 👷 Todo
```
Comment (rymaju) [inline README.md:8]: Maybe we should delete the ##Todo section too?

File: README.md [hunk 3]
```diff
@@ -100,40 +132,17 @@ Work in Progress, will finish when we solidify our project structure.
...
-
-To learn more about Next.js, take a look at the following resources:
+## 🔧 Troubleshooting
 
-- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
-- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
+Various commands that may help in the development process:
 
-You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
+| Command | Description |
```
Comment (rymaju) [inline README.md:139]: 😍 So pretty!

**Review by rymaju — APPROVED** [review]
I left some feedback but it meets the acceptance criteria so LGTM ⭐

File: README.md (hunk 3 above)
Comment (surbhi-gulati) [reply README.md:139]: :D

File: README.md (hunk 2 above)
Comment (surbhi-gulati) [reply README.md:8]: sgtm, removing - can we take note somewhere else unless that URL is final?

File: README.md (hunk 1 above)
Comment (surbhi-gulati) [reply README.md:120]: done

**Review by rymaju — DISMISSED** [review]
LGTM 💯

File: README.md [hunk 4]
```diff
@@ -1,6 +1,8 @@
 # 🏡 Virtual Senior Center
 
-A project to create a virtual community for LGBT seniors in the Massachusetts area.
+The mission of the outstandinglife.org is to improve the quality of life of LGBTQ older adults in Massachusetts. Through this project, C4C aims to help LBFE find new creative ways to connect older adults: engaging hearts, transforming lives and breaking down boundaries.
```
Comment (varunthakkar1) [inline README.md:3]: 👍

File: README.md [hunk 5]
```diff
@@ -62,9 +67,23 @@ npm run test:e2e      # runs all E2E tests in headless mode
 npm run test:cy       # opens the Cypress app to run E2E tests visually
 ```
 
-## Project Structure
-
-Work in Progress, will finish when we solidify our project structure.
+## 📂 Project Structure
+
+There are four main folders in this project:
```
Comment (varunthakkar1) [inline README.md:72]: 🔧 Add the lib folder to this as well. It contains frontend components and most importantly, helpers/DTOs for the API routes!

File: README.md [hunk 6]
```diff
@@ -100,40 +124,17 @@ Work in Progress, will finish when we solidify our project structure.
...
-
-To learn more about Next.js, take a look at the following resources:
+## 🔧 Troubleshooting
 
-- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
-- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
+Various commands that may help in the development process:
 
-You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
+| Command | Description |
```
Comment (varunthakkar1) [inline README.md:131]: ⛏ add 'npx prisma studio' to this because it is helpful

**Review by varunthakkar1 — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (surbhi-gulati) [reply README.md:131]: done

File: README.md (hunk 5 above)
Comment (surbhi-gulati) [reply README.md:72]: done

**Review by rymaju — DISMISSED** [review]
LGTM!!! 💯

**Review by varunthakkar1 — APPROVED** [review]

**Pull request merged**